main.css

```diff
@@ -0,0 +1,88 @@
+header {
+  background-color: #3c3a39;
+  color: white;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+  position: absolute;
+  width: 100%;
+  left: 0;
+  top: 0;
+}
+
+.navbar {
+  align-items: flex-end;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-left: 30px;
+}
+
+ul {
+  display: inline-flex;
+}
+
+h1 {
+  color: #ff6b00;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 56px;
+  line-height: 64px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+body {
+  background-image: url(./images/Illustration.svg);
+  background-color: #1c1a19;
+  background-repeat: no-repeat;
```

Comment:
- [ ] [OPTIONAL] Kindly consider adding background images to the appropriate section, which is in this case the headline section.